docs/conf.py

```diff
@@ -47,7 +47,6 @@
 from searx.webapp import application
```

Comment:
The `application` import is now unused. Could you please add a comment to explain that this import is necessary to initialize the plugins?

Reply:
done

Reply:
Ah yeah I think changing the import to `import searx.webapp` would be better (since the application isn't needed in the scope).

Reply:
opps sorry / done